src/components/CategoriesPage.js

```diff
@@ -37,6 +43,7 @@ function Employees(props) {
 
   return (
     <Container style={{ textAlign: "left" }} className="Employees">
+      <SearchBar {...props} handleSubmit={(query) => handleSubmit(query)}  />
```

Comment:
I dont see you use ** handleSubmit** prop in the SearchBar

Reply:
Done.